core/internal/cltest/job_factories.go

```diff
@@ -2,14 +2,18 @@ package cltest
 
 import (
 	"fmt"
+
 	"testing"
 
+	"github.com/smartcontractkit/chainlink/core/internal/testutils/evmtest"
+	"github.com/smartcontractkit/chainlink/core/logger"
+	"github.com/smartcontractkit/sqlx"
```

Comment:
AFAIK we should be sorting imports so that all smartcontractkit/* are grouped in one section